publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategyTest.java

```diff
@@ -176,7 +179,8 @@ void shouldDenyCuratorOnDegreeWithNoResourceOwnerAffiliation(PublicationOperatio
 
     @ParameterizedTest(name = "Should allow Curator {0} operation on degree resources with matching resource owner "
                               + "affiliation")
-    @EnumSource(value = PublicationOperation.class, mode = Mode.EXCLUDE, names = {"DELETE", "TERMINATE"})
+    @EnumSource(value = PublicationOperation.class, mode = Mode.EXCLUDE,
+        names = {"DELETE", "TERMINATE", "UPDATE_FILES"})
     void shouldAllowCuratorOnDegreeWithResourceOwnerAffiliation(PublicationOperation operation)
         throws JsonProcessingException, UnauthorizedException {
```

Comment:
"Curator" should not be able to edit published files, only publishing curator can do it